README.md.tt

```diff
@@ -45,7 +45,7 @@ foreman start -f Procfile.dev
 
 Add the following build settings to run the tests in the Docker environment via Docker Compose (configuration in `docker-compose.test.yml`):
 
-* Configure the environment variable `BRANCH_TAG` to tag Docker images per branch:
+* Configure the environment variable `BRANCH_TAG` to tag Docker images per branch, e.g:
 
 ```sh
 export BRANCH_TAG=$SEMAPHORE_BRANCH_ID
```

Comment:
Should we change the `SEMAPHORE_BRANCH_ID` here @longnd

Reply:
Or in case you think this part is redundant as we have `semaphore or github action` setup already, then we can remove this part.

Reply:
I made the change in b4f8911 to clarify which value should be passed to BRANCH_TAG env variable. I prefer to keep the current content of README since it mentions both Github Action & SemaphoreCI. It also clarify the setup behind the Docker-based test for CI. What do you think?